src/components/banner/banner.test.js

```diff
@@ -36,30 +36,29 @@ describe('banner elements', () => {
 });
 
 describe('banner events', () => {
-  test('should change image position on mousemove once every 100ms', () => {
+  test('should change image position on mousemove once every 100ms', async () => {
     const { getByAltText } = render(<Banner />);
     const targetElem = getByAltText('img');
+    jest.useFakeTimers();
```

Comment:
investigate this line

Reply:
If remove this line it will cause new warning to appear on npm run test:coverage